onchain/rollups/contracts/consensus/authority/Authority.sol

```diff
@@ -30,12 +30,6 @@ contract Authority is AbstractConsensus, Ownable {
     /// @dev See the `getHistory` and `setHistory` functions.
     IHistory internal history;
 
-    /// @notice The `Authority` contract was created.
-    /// @param owner The address that initially owns the `Authority` contract
-    /// @param inputBox The input box from which the authority fetches new inputs
-    /// @dev MUST be triggered during construction.
-    event ConsensusCreated(address owner, IInputBox inputBox);
```

Comment:
you can also remove the import of `IInputBox`. Line 20

Reply:
Hey, you're right! Thanks for spotting this. :-)